spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -931,4 +931,15 @@ def fixture_record(fixture_name, indexer: @indexer)
       expect(sample).to be_nil
     end
   end
+
+  context "Temporary in resource sharing location" do
+    it "does not show as a temporary location" do
+      indexer = IndexerService.build
+      sample = indexer.map_record(fixture_record('998370993506421', indexer: indexer))
+      holdings = JSON.parse(sample['holdings_1display'][0])
+      expect(holdings['22561746630006421']['items'].count).to eq 1
+      expect(holdings['22561746630006421']["location"]).to eq "Remote Storage (ReCAP): Mendel Music Library Use Only"
```

Comment:
@christinach I assume from your comment in slack this is the wrong value.  Could you please tell me what value you are expecting?  It seems to match with your earlier comment.

Reply:
@carolyncole I don't understand your question. I didn't say in slack that this is the wrong value. You are returning what I added in my previous comment. The mocked up holdings_1display. Which looks good to me.